### PR TITLE
root: Add version 6.20.*, use *.08

### PR DESCRIPTION
**Pull request by ChristianTackeGSI**
* Simplify some option handling for builtins
* Get all the 6.20.* versions into package.py
* Use 6.20.08 for the upcoming release.

### CODE REVIEW

**Comment by ChristianTackeGSI** [comment]
@MohammadAlTurany, @kresan, @TobiasStockmanns

I have added all the 6.20 releases of root to the spack recipe (00, 02, 04, 06, 08). I tested a selection (02, 06, 08) and they seem to work fine (compile, run our FairSoft test suites).

I have selected 6.20.08 for the upcoming release for now.
That's this PR: CI just finished.
Is that okay with all of you? Or would you rather see an older version?

**Comment by kresan** [comment]
I am fine with 6.20.08.
Adding @fuhlig1 for his answer for CBM.

**Comment by fuhlig1** [comment]
I did not test myself but I don’t see any problems with the new version.

**Comment by ChristianTackeGSI** [comment]
@TobiasStockmanns confirmed, that he's okay with the latest ROOT.

The failing test seems to be a workflow one. I am not 100% sure, that it is root related. I doubt it. Because the CI worked out before.

**Review by dennisklein — APPROVED** [review]

File: repos/fairsoft/packages/root/package.py [hunk 1]
```diff
@@ -313,28 +321,31 @@ def cmake_args(self):
...
+        builtin_opts\
+            = [
+                ['builtin_llvm', True],
+                ['builtin_afterimage', True],
+                ['builtin_cfitsio', False],
+                ['builtin_davix', False],
+                ['builtin_fftw3', False],
+                ['builtin_freetype', False],
+                ['builtin_ftgl', False],
+                ['builtin_gl2ps:BOOL', True],
```
Comment (dennisklein) [inline repos/fairsoft/packages/root/package.py:333]:
```suggestion
                ['builtin_gl2ps', True],
```

**Pull request merged**